### ISSUE TITLE
Region-aware Adaptive Instance Normalization for Image Harmonization

### ISSUE THREAD

**Issue by reyllama** [reyllama/paper-reviews#118]
*CVPR 2021*

# TL;DR

- Image Harmonization: compose a foreground object to existing background image in a natural way
- RAIN: AdaIN adapted for foreground-background style transfer

# Motivation

- Traditional image harmonization approaches relied on either hand-crafted features or directly learning composed image

# Previous Work

- Image Harmonization is a ***blue ocean***
- Style Transfer
- Normalization (BN, IN, AdaIN, SPADE)

# Method

![image](https://user-images.githubusercontent.com/46308407/131235468-6c1730e4-3bfb-45e2-aa28-703012afba89.png)
- Add normalization layers to U-Net.
![image](https://user-images.githubusercontent.com/46308407/131235474-4fb111ea-c44b-49c3-a9da-26bd05ea63ac.png)
- Instead of transferring channelwise statistics from a style image to a content image (*AdaIN*), transfer statistics of background to foreground.
- Instance Norms in encoder and **R**egion-Aware **A**daptive **I**nstance **N**ormalization in decoder.
- 3 Attention blocks in decoder (as done in relevant previous works)

# Experiments

## Dataset
- iHarmony4: HCOCO, HAdobe5K, HFlickr, Hday2night

## Metric
- PSNR, MSE, fMSE

![image](https://user-images.githubusercontent.com/46308407/131235531-2ca4d022-9511-4654-8e21-459e4c455fde.png)
![image](https://user-images.githubusercontent.com/46308407/131235532-44dfc78e-a7cb-4ac5-8d80-6f3e2ea57893.png)
![image](https://user-images.githubusercontent.com/46308407/131235535-a6b3a443-6af5-4734-bd4c-ee83e5f5580f.png)
![image](https://user-images.githubusercontent.com/46308407/131235540-464838b0-cc69-4a50-8f75-b17ae6a98b5c.png)
![image](https://user-images.githubusercontent.com/46308407/131235544-c43aef2b-16d0-462b-8ed6-7fb5ae695e80.png)
![image](https://user-images.githubusercontent.com/46308407/131235551-133c46c3-44be-4426-8740-e6ae23dcffe7.png)
![image](https://user-images.githubusercontent.com/46308407/131235565-b6c7c2a7-b1db-411c-afa2-ec8a1c3a2332.png)

# Comments
- Finding a less explored task is also an important talent as a researcher.
- Very simple, yet quantitative results are quite favorable. Wonder how has the baselines been in this task.
